apps/glusterfs/operations_block_volume.go

```diff
@@ -431,7 +459,20 @@ func (vdel *BlockVolumeDeleteOperation) Finalize() error {
 			return e
 		}
 
-		vdel.op.Delete(tx)
-		return nil
+		return vdel.op.Delete(tx)
```

Comment:
So this is a difference in behavior of the `Finalize()` function in that now error of the `Delete` op will result in failure of `Finalize`. Is this intended? Was that an oversight before? What has changed?

Reply:
It was an oversight before.